Sources/UIApplication+handleSDLEvents.swift

```diff
@@ -93,6 +93,28 @@ extension UIApplication {
                 UIApplication.onWillEnterForeground()
             case SDL_APP_DIDENTERFOREGROUND:
                 UIApplication.onDidEnterForeground()
+            case SDL_WINDOWEVENT:
+                let windowEventId = SDL_WindowEventID(UInt32(e.window.event))
+
+                guard windowEventId == SDL_WINDOWEVENT_RESIZED else {
+                    break
+                }
+
+                #if os(Android)
+                let newWidth = CGFloat(e.window.data1) / UIScreen.main.scale
```

Comment:
I think it'd be worth mentioning (in a code comment) that this matches how we init `UIScreen` on Android, due to quirks in SDL on that platform. Basically, we init UIScreen in @1x pixels on Android, but in (potentially) retina _points_ on Mac (etc.)